dask/tests/test_delayed.py

```diff
@@ -101,6 +101,14 @@ def test_operators():
     assert (a >> 1).compute() == 5
     assert (a > 2).compute()
     assert (a ** 2).compute() == 100
+    if PY3:
+        class dummy:
+            def __matmul__(self, other):
+                return 4
+        c = delayed(dummy())
+        d = delayed(dummy())
+
+        assert (c @ d).compute() == 4
```

Comment:
If this causes a SyntaxError in Python 2 (seems likely)  then we might wrap this in a string and then call `eval` like the following:

```python
In [1]: x = 1 

In [2]: eval('x + x')
Out[2]: 2
```

Reply:
Hmm, you are right. Ok, let me try again
https://travis-ci.org/dask/dask/jobs/398449158#L1338